docs/getting-started/configure-integration-policy.asciidoc

```diff
@@ -169,7 +169,7 @@ Malicious behavior protection levels are:
 * **Prevent** (Default): Detects malicious behavior on the host, forces the process to stop,
 and generates an alert.
 
-Select whether you want to use **Reputation service** for additional malware analysis. This service identifies malicious activity and false positives, and enriches alerts using data from various sources, such as VirusTotal and telemetry. For example, reputation service can detect suspicious downloads of binaries with low or malicious reputation.
+Select whether you want to use **Reputation service** for additional protection. Elastic's **Reputation service** leverages our extensive threat intelligence knowledge to make high confidence real-time block decisions. For example, reputation service can detect suspicious downloads of binaries with low or malicious reputation.
```

Comment:
No need to capitalize or bold the term when not referring directly to the UI. Also, would "prevention decisions" be an OK replacement for "block decisions"? It might be a little more consistent with other language in the docs.
```suggestion
Select whether you want to use **Reputation service** for additional protection. Elastic's reputation service leverages our extensive threat intelligence knowledge to make high confidence real-time prevention decisions. For example, reputation service can detect suspicious downloads of binaries with low or malicious reputation.
```

Reply:
both work for me!